Client/src/Modelo/main.java

```diff
@@ -0,0 +1,28 @@
+package Modelo;
+
+public class main
+{
+
+	public static void main(String[] args)
+	{
+		Torneo nuevo_torneo = Torneo.getInstance();
```

Comment:
```suggestion
		Torneo torneo = Torneo.getInstance();
```

Para respetar la nomenclatura determinada de las variables en Java. En este caso es irrelevante ponerle a la variable "nuevo" ya que únicamente existirá un solo torneo el que estaremos creando.